playground/NodeEditor.js

```diff
@@ -575,49 +317,95 @@ export class NodeEditor extends EventDispatcher {
 
 			const filename = button.getExtra();
 
-			const loader = new Loader( Loader.OBJECTS );
-			const json = await loader.load( `./jsm/node-editor/examples/${filename}.json`, ClassLib );
-
-			this.loadJSON( json );
+			this.loadURL( `./examples/${filename}.json` );
 
 		};
 
-		const addExample = ( context, name, filename = null ) => {
+		const addExamples = ( category, names ) => {
+
+			const subContext = new ContextMenu()
 
-			filename = filename || name.replaceAll( ' ', '-' ).toLowerCase();
+			for( const name of names ) {
```

Comment:
Here a space is missing between for and left bracket (and i think same thing happens in other places too).